### DISCUSSION TITLE
Designer shows only transparent window

### DISCUSSION THREAD

**Discussion by ljuuset** [AvaloniaUI/Avalonia#9661] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/120189145/206754358-be784e37-8f77-446e-9100-f0deadf5cd25.PNG)
The designer window only shows this transparent window inside of it after starting a new project and building. Suggestions on what this could be about?

**Comment by timunie** [comment, marked as answer]
As far as I can see there is no content at all in your Window. So nothing to show. The previewer is unable to preview the system chrome. So it will only show the content.
